### DISCUSSION TITLE
Storybook: Promote Usage Examples

### DISCUSSION THREAD

**Discussion by tveinfeld** [Vonage/vivid#1060] [General]
A few suggestions regarding Storybook's content layout that popped into my mind while reviewing #1057:
 
 * I think we should position "Templates" above "Core",  perhaps even above "Introduction" (when there are enough examples to show).
Another option would be to promote "Getting Started" to a section of its own, and include "Templates" within it.
 * Rebrand "Templates" as "Usage Examples" (or something more direct)
 * Add a code panel alongside the demoed examples

**Comment by yinonov** [comment]
Good ideas, especially "Usage Examples" within "Getting Started".
had trouble deciding where in the repo to place the actual story as it is a mixture of components. didn't want to just lay it out on the root.
there's also a the [showcase website](/Vonage/vivid-astro) I thought we can migrate to to avoid such decisions 👆

**Reply by tveinfeld** [reply]
even better... storybook is a "heavy", some users would like the option to have a different venue for getting all their example snippets from 👍 

can we publish this site on its own domain and link to it from sb?

**Reply by yinonov** [reply]
The objective is to drop storybook altogether

**Reply by tveinfeld** [reply]
perhaps we should take a "gradual transition" strategy where all the usage samples are housed in this new site which Storybook will refer users to, and then gradually enrich the content available on it until SB is no longer relevant?

**Reply by yinonov** [reply]
Sure, let's try focusing on that this coming sprint
